Workbooks/WorkbooksMetadata.json

```diff
@@ -1146,11 +1146,11 @@
     "logoFileName": "Darktrace.svg",
     "description": "A workbook containing relevant KQL queries to help you visualise the data in model breaches from the Darktrace Connector",
     "dataTypesDependencies": [ "CommonSecurityLog" ],
-    "dataConnectorsDependencies": [],
-    "previewImagesFileNames": [ "DarktraceSummaryWhite.png", "DarktraceSummaryBlack.png" ],
-    "version": "1.0",
-    "title": "Darktrace Model Breach Summary",
-    "templateRelativePath": "Darktrace.json",
+    "dataConnectorsDependencies": [ "AIA-Darktrace" ],
```

Comment:
dataConnectorsDependencies matches the data connector id, could you please update the dependencies as well.

Thanks,
Aditi Maheshwari